src/components/DiffingResult/index.jsx

```diff
@@ -60,39 +66,163 @@ function DiffingResult() {
+    setFrameName(ev.target.getAttribute("data-name"));
+  };
+
   useEffect(() => {
-    if (diffingResult) {
-      if (diffingResult.result === "error") {
-        setToast({ status: true, message: diffingResult.message });
+    const initCanvas = () => {
+      const canvasInit = new fabric.Canvas("canvas", {
+        width: window.innerWidth,
+        height: window.innerHeight,
+        backgroundColor: "#CED4DA",
+        setZoom: 0.3,
+        selection: false,
+      });
+
+      return canvasInit;
+    };
+
+    const newCanvas = initCanvas();
+    newCanvas.on("mouse:wheel", ev => {
+      const delta = ev.e.deltaY;
+      let Zoom = canvasRef.current.getZoom();
+
+      Zoom *= 0.999 ** delta;
+      Zoom = Math.max(0.01, Math.min(20, Zoom));
+
+      canvasRef.current.zoomToPoint({ x: ev.e.offsetX, y: ev.e.offsetY }, Zoom);
+
+      ev.e.preventDefault();
+      ev.e.stopPropagation();
+    });
+
+    canvasRef.current = newCanvas;
+
+    const resizeCanvas = () => {
+      if (canvasRef.current) {
+        canvasRef.current.setHeight(window.innerHeight - 90);
+        canvasRef.current.setWidth(window.innerWidth - 290);
+
+        canvasRef.current.calcOffset();
+        canvasRef.current.renderAll();
+      }
+    };
+
+    window.addEventListener("resize", resizeCanvas);
 
-        return;
+    const fetchImageURLToFigma = async () => {
+      const imgUrlObject = await fetchImageUrl(projectKey, setToast);
+
+      setImageUrl(imgUrlObject);
+    };
+
+    if (!imageUrl) {
+      fetchImageURLToFigma();
+    }
+
+    return () => {
+      window.removeEventListener("resize", resizeCanvas);
+
+      if (canvasRef.current) {
+        canvasRef.current.dispose();
       }
+    };
+  }, []);
+
+  useEffect(() => {
+    if (frameList.length > 0) {
+      const firstFrame = frameList[0];
+
+      setFrameId(firstFrame.id);
+      setFrameName(firstFrame.name);
+    }
+  }, [frameList]);
 
-      const frames = [];
+  useEffect(() => {
+    if (!diffingResult) {
+      return;
+    }
 
-      for (const frameId in diffingResult.content.frames) {
+    if (diffingResult.result === "error") {
+      setToast({ status: true, message: diffingResult.message });
+
+      return;
+    }
+
+    const frames = [];
+
+    for (const frameId in diffingResult.content.frames) {
+      if (
+        diffingResult.content.frames.hasOwnProperty.call(
+          diffingResult.content.frames,
+          frameId,
+        )
+      ) {
         const frame = diffingResult.content.frames[frameId];
 
         frames.push({ name: frame.name, id: frameId });
       }
-
-      setFrameList(frames);
     }
+
+    setFrameList(frames);
   }, [diffingResult]);
 
   useEffect(() => {
-    const newCanvas = new fabric.Canvas("canvas", {
-      width: 650,
-      height: 650,
-      backgroundColor: "#CED4DA",
-    });
+    const fetchImageURLToFigma = async () => {
+      const imgUrlObject = await fetchImageUrl(projectKey, setToast);
 
```

Comment:
image 부분만 반영 부탁드립니다~!

```suggestion
    const fetchImageURLToFigma = async () => {
      const imageUrlObject = await fetchImageUrl(projectKey, setToast);

```